src/Symfony/Component/Console/Tests/Helper/DescriptorHelperTest.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Tests\Helper;
+
+use Symfony\Component\Console\Helper\DescriptorHelper;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class DescriptorHelperTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @expectedException        \InvalidArgumentException
+     * @expectedExceptionMessage Unsupported format "invalid".
+     */
+    public function testDescribeWithInvalidFormat()
+    {
+        $descriptor = new DescriptorHelper();
+        $descriptor->describe($this->getMock(OutputInterface::class), null, array('format' => 'invalid'));
+    }
+
+    public function testGetName()
+    {
+        $descriptor = new DescriptorHelper();
+
+        $this->assertSame('descriptor', $descriptor->getName());
```

Comment:
>is null object 

$descriptor = new DescriptorHelper(); ? 

the name will always be set, so the test is valid to make sure of that to prevent regression
https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Console/Helper/DescriptorHelper.php#L92